config/environments/production.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'active_support/core_ext/integer/time'
```

Comment:
was the inclusion of this `require` in prod and test configs but not the dev config intentional, or an oversight?  i'm not totally sure what extensions this provides, but a bit of cursory searching indicates that making the same inclusion in all envs is advised? https://github.com/rails/rails/commit/c8c549750da40cd540ad1064b0b75e5b4e32d3f1

Reply:
Yes, I think this was a merge problem. I'll add that to development.rb too.